src/lib/endpoints.ts

```diff
@@ -1,39 +1,43 @@
-import { CHAIN_ID } from '@/lib/constants'
+import { CHAIN_ID, EVENTS_FROM_BLOCK } from '@/lib/constants'
+
+const CHAIN_ID_PARAM = `chainId=${CHAIN_ID}`
+const FROM_BLOCK_PARAM = `fromBlock=${EVENTS_FROM_BLOCK}`
 
 export const fetchAddressTokensEndpoint =
-  process.env.NEXT_PUBLIC_API_RWS_TOKEN_BY_ADDRESS || '/address/{{address}}/tokens?chainId={{chainId}}'
+  process.env.NEXT_PUBLIC_API_RWS_TOKEN_BY_ADDRESS ||
+  `/address/{{address}}/tokens?chainId={{chainId}}&${FROM_BLOCK_PARAM}`
```

Comment:
These events are managed by the DAO, so I don't think we should use the CR related block. I think we could just leave them as they are.

Reply:
yea that's what I was thinking...tx for the confirmation